home/models.py

```diff
@@ -448,7 +448,11 @@ class BannerPage(Page):
 
 class FooterIndexPage(Page):
     parent_page_types = ['home.HomePage']
-    subpage_types = ['home.FooterPage']
+    subpage_types = ['home.FooterPage', 'home.PageLinkPage']
```

Comment:
Could you add support for the other page types (article, survey, poll, quiz) as well?

Reply:
done